Assets/Scripts/UI/Menus/AudioOptionsMenu.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.UI;
+
+namespace OperationBlackwell.UI {
+	public class AudioOptionsMenu : Menu {
+
+		[SerializeField] private Slider backgroundMusicSlider_;
+		[SerializeField] private Slider soundEffectSlider_;
+		[SerializeField] private AudioMixer backgroundMusicMixer_;
+		[SerializeField] private AudioMixer soundEffectMixer_;
+
+		private void Start() {
+			UpdateSliders();
```

Comment:
Are the Sliders only updated on start??

Reply:
Nope, but this forces them to be whatever the current value the mixer has on scene load.

Reply:
Okay, but how are the sliders updated then?

Reply:
By moving them? It's just an unity slider.

Reply:
Im just stupid leave me alone........